tools/cldr/lib/install-cldr-jars.sh

```diff
@@ -97,19 +92,6 @@ run_with_logging mvn -B install:install-file \
   -DlocalRepositoryPath=. \
   -Dfile="${CLDR_TOOLS_DIR}/cldr.jar"
 
-echo "Installing CLDR utilities JAR file..."
```

Comment:
Not needing this utilities JAR is a requirement for expanding who can use the CLDR API in ICU (since the code in this utility JAR comes from ICU originally and creates the potential for duplicate (incompatible) classes at build time.